js/editor-libs/console-utils.js

```diff
@@ -19,7 +19,9 @@ module.exports = {
                 output += this.formatArray(input[i]);
                 output += ']';
             } else {
-                output += this.formatObject(input[i]);
+              if (input[i] !== undefined) {
```

Comment:
Should probably just turn this into an `if else {` instead of the nested `if` inside the `else`